sass/true/_modules.scss

```diff
@@ -32,7 +32,15 @@
 ) {
   @include _true-reset(module);
   @include _true-context(module, $name);
-  @include _true-message('# #{$name} ----------', comments);
+  @include _true-message('# Module: #{$name}', 'comments');
+
+  $underline: '----------';
+
+  @for $i from 1 through str-length($name) {
+    $underline: '#{$underline}-';
```

Comment:
If I were using it anywhere else, I'd agree. But this code is short, never repeated, and doesn't require extra arguments. I'd be ok with the split, but I don't see any gain.